src/dvclive/huggingface.py

```diff
@@ -14,11 +15,33 @@
 
 
 class DVCLiveCallback(TrainerCallback):
-    def __init__(self, model_file=None, live: Optional[Live] = None, **kwargs):
+    def __init__(
+        self,
+        live: Optional[Live] = None,
+        log_model: Optional[Literal["all", "last"]] = None,
+        **kwargs,
+    ):
         super().__init__()
-        self.model_file = model_file
+        self._log_model = log_model
         self.live = live if live is not None else Live(**kwargs)
 
+    def on_train_begin(
+        self,
+        args: TrainingArguments,
+        state: TrainerState,
+        control: TrainerControl,
+        **kwargs,
+    ):
+        for key, value in args.to_dict().items():
+            if key in (
+                "num_train_epochs",
+                "weight_decay",
+                "max_grad_norm",
+                "warmup_ratio",
+                "warmup_steps",
+            ):
+                self.live.log_param(key, value)
```

Comment:
I know it feels like a lot of unnecessary params, but this goes against what all the other loggers do, right? I think we would need a really strong reason to do this, and IMHO logging dozens of parameters is typical and unproblematic -- even if I probably never care about a parameter, once in awhile I might want to know what it was and it's better to have than not.